pages.ru/common/cwebp.md

```diff
@@ -0,0 +1,24 @@
+# cwebp
+
+> Конвертирует файл изображения в формат WebP.
+> Больше информации: <https://developers.google.com/speed/webp/docs/cwebp>.
+
+- Сжатие WebP со стандартными настройками (качество = 75) в сохранением в выходной ([o]utput) файл:
+
+`cwebp {{путь/к/изображению}} -o {{путь/к/результату.webp}}`
+
+- Сжатие WebP с наилучшим качеством ([q]uality) и наибольшим размером файла:
+
+`cwebp {{путь/к/изображению}} -o {{путь/к/результату.webp}} -q {{100}}`
+
+- Сжатие WebP с наихудшим качеством и наименьшим размером файла:
+
+`cwebp {{путь/к/изображению}} -o {{путь/к/результату.webp}} -q {{0}}`
+
+- Сжатие WebP с изменением размера изображения:
```

Comment:
```suggestion
- Сжать WebP, изменив размер изображения:
```

Reply:
I don't like this variant. It results into illiterate sentence. Russians can understand it but it's not Russian.

I propose to change the first part of the sentence and leave the second as is.